routes/auth.js

```diff
@@ -43,12 +43,12 @@ router.route("/login")
             password: req.body.password
         };
         authHelper.authenticate(req.body.email, req.body.password).then(function (user) {
-            if (!user) {
-                return res.status(401).send({error: "Incorrect username/password"});
+            if (user.error === 406) {
+                return res.status(406).send(user)
```

Comment:
Wait, with incorrect password you send back the user?!

Reply:
No i dont, its just what the parameter is called... I also found that confusing... could rename it to result or something? The user object in this case is something like {error: 406, data: "wrong password"}

Reply:
Ah okay